docs/ConfigureScalarDL.md

```diff
@@ -207,17 +134,19 @@ If you have not created the `scalardl` Kubernetes secret, you can use the follow
 auditor:
   ....
   scalarAuditorConfiguration:
-    dbStorage: dynamo
-    dbContactPoints: <REGION>
-    dbUsername: <AWS_ACCESS_KEY_ID>
-    dbPassword: <AWS_ACCESS_SECRET_KEY>
+    dbContactPoints: <contactPoints>
+    dbUsername: <username>
+    dbPassword: <password>
+    dbStorage: <storageType>
 ```
 
-To enable the Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
+Note:-
```

Comment:
```suggestion
Note:
```

Reply:
I have corrected the note title to  `Note:`  in all places in which it is used.
Thank you